### PR TITLE
feat(colors): Adds BEHR paint colors

### PR DESCRIPTION
**Pull request by cheesits456**
Added colors suggested in issue #86

### CODE REVIEW

File: dist/colornames.csv [hunk 1]
```diff
@@ -250,11 +285,13 @@ Algal Fuel,#21c36f
 Algen Gerne,#479784
 Algerian Coral,#fc5a50
 Algiers Blue,#00859c
+Algod0n Azul,#c1dbec
```
Comment (meodai) [inline dist/colornames.csv:288]: Is this meant to be a 0 ?

File: dist/colornames.csv [hunk 2]
```diff
@@ -5061,6 +6085,7 @@ Deadly Depths,#111144
 Deadnettle White,#d2dad0
 Deadsy,#c2a84b
 Deadwind Pass,#596d7f
+Death By Chocolate,#60443f
```
Comment (meodai) [inline dist/colornames.csv:6088]: Shoud be "Death by Chocolate"

File: dist/colornames.csv [hunk 3]
```diff
@@ -5039,11 +6059,15 @@ Day Glow,#eadd82
...
 Day On Mercury,#d5d2d1
+Day Spa,#eaefed
 Daybreak,#8981a0
+Daybreak Sun,#f7eecb
 Daydream,#e3ebae
 Daydreaming,#f4f0e1
 Dayflower,#38a1db
+Daylight Lilac,#a385b3
 Daystar,#fff8da
+Dazzle And Delight,#d99b7b
```
Comment (meodai) [inline dist/colornames.csv:6070]: Dazzle and Delight

File: dist/colornames.csv [hunk 4]
```diff
@@ -4996,32 +6007,40 @@ Dark Sorrel,#587a65
...
 Darkshore,#464964
 Darlak,#4f4969
+Darlin Clementine,#d29f7a
 Darling Bud,#ff88ff
 Darling Lilac,#c9acd6
 Darth Vader,#0b0b0b
 Dartmoor Mist,#cddce3
 Dartmouth Green,#00703c
+Dash Of Curry,#ca6e5f
+Dash Of Oregano,#928459
```
Comment (meodai) [inline dist/colornames.csv:6043]: of: https://johnresig.com/files/titleCaps.js ?

File: dist/colornames.csv [hunk 5]
```diff
@@ -4907,15 +5900,25 @@ Daphne Rose,#c37cb3
 Dapper Dingo,#e2c299
 Dapper Greyhound,#697078
 Dapple Grey,#959486
+Dappled Sunlight,#f2e3c9
+Daredevil,#ab4343
 Daring Indigo,#374874
 Dark,#1b2431
 Dark & Stormy,#353f51
 Dark Ages,#9698a3
+Dark As Night,#495252
```
Comment (meodai) [inline dist/colornames.csv:5909]: as

File: dist/colornames.csv [hunk 6]
```diff
@@ -9498,10 +11453,14 @@ Key Largo,#7fb6a4
 Key Lime,#aeff6e
 Key Lime Pie,#bfc921
 Key Lime Water,#e8f48c
+Key To The City,#bb9b7c
```
Comment (meodai) [inline dist/colornames.csv:11456]: Key to the City

File: dist/colornames.csv [hunk 7]
```diff
@@ -9498,10 +11453,14 @@ Key Largo,#7fb6a4
 Key Lime,#aeff6e
 Key Lime Pie,#bfc921
 Key Lime Water,#e8f48c
+Key To The City,#bb9b7c
+Keystone,#b39372
+Keystone Gray,#b6bbb2
```
Comment (meodai) [inline dist/colornames.csv:11458]: Grey

File: dist/colornames.csv [hunk 8]
```diff
@@ -19922,44 +23934,68 @@ Wingsuit Wind,#bad5d4
...
+Winter Fresh,#d6eedd
 Winter Frost,#e4decd
+Winter Garden,#dcc89d
+Winter Glaze,#eaebe0
 Winter Haven,#e1e6eb
+Winter Haze,#cec9c1
 Winter Hazel,#d0c383
+Winter Hedge,#798772
+Winter Ice,#dbdfe9
+Winter In Paris,#6e878b
```
Comment (meodai) [inline dist/colornames.csv:23966]: Winter in Paris

**Comment by meodai** [comment]
@cheesits456 https://github.com/meodai/color-names/issues/86#issuecomment-662146571 I think you introduced the capitals though.

**Comment by cheesits456** [comment]
oh right, yeah, I wrote a function called `capitalize()` and ran all the name values through it so the capitalization formatting would match the rest of the list, but again, it was all done programmatically, no manual editing :P 
```js
function capitalize(text) {
	let r = text.toLowerCase().replace(/(?<=\s)\w/g, c => c.toUpperCase());
	return r.replace(/^\w/, c => c.toUpperCase());
}
```

**Comment by meodai** [comment]
@cheesits456 will you fix the capitalization? I can also do it.

**Comment by cheesits456** [comment]
I got a lot of stuff going on rn, if you don't mind then by all means, go ahead 👍

**Comment by meodai** [comment]
@cheesits456 its in. Thanks! was the most complex interactive merge I ever did :D thank god for `git add . -p`

**Pull request merged**